desktop/ui/src/test/java/org/datacleaner/widgets/result/ValueDistributionResultSwingRendererGroupDelegateTest.java

```diff
@@ -28,100 +28,97 @@
 import org.datacleaner.result.ValueCountListImpl;
 import org.datacleaner.storage.RowAnnotationImpl;
 
-public class ValueDistributionResultSwingRendererGroupDelegateTest extends
```

Comment:
I think there are no changes in the test, depite the diff. Are we covered properly by test?